src/applications/simple-forms/form-upload/containers/EditLink.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+import { VaLink } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+
+const EditLink = ({ href, router }) => {
+  const [link, setLink] = useState(null);
+
+  useEffect(
```

Comment:
can get rid of the useEffect and useState I think and just set the href directly.